mst/backup-types.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Types of Backups
+excerpt: Understand how Timescale Cloud backs up PostgreSQL
+product: cloud
+keywords: [backup, binary, logical]
+tags: [backup, binary, logical]
```

Comment:
```suggestion
keywords: [backup]
tags: [backup, binary backup, logical backup]
```
You could also keep binary and logical in the keywords if you prefer (keywords get show, tags are invisible and only help rank related content)